sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -611,14 +611,26 @@ abstract class HadoopFsRelation private[sql](maybePartitionSpec: Option[Partitio
       } else {
         rdd.asInstanceOf[RDD[InternalRow]]
       }
+
```

Comment:
can you update the comment for this function to explain this will incur conversions 3 times?